config/locales/en.yml

```diff
@@ -161,10 +161,10 @@ en:
           <a href="http://www.enable-javascript.com/">internet browsers with JavaScript</a>.
       offline:
         call_to_action: Offline
-        description: Web chat is available from %{hours}.
```

Comment:
I believe removing this here is the issue

Reply:
If the requirement for outside availability hours, is to show what hours the chat line is available, there is no need to change line 164 in the translation file. [this bit of code](https://github.com/moneyadviceservice/frontend/blob/master/app/decorators/chat_opening_hours_decorator.rb#L46-L52) adds the hours to the view `(shared/_contact_panels)`

Reply:
appreciate the comments but heres why the changes are set this way...
<img width="448" alt="screen shot 2018-07-17 at 21 02 16" src="https://user-images.githubusercontent.com/10046127/43141903-043a2b28-8f4f-11e8-84f5-f6bdbec716b8.png">
it leverages the webchat hours defined in the CMS :)

Reply:
Can we then change it to be "Web chat is currently unavailable" or something along those lines?

Reply:
Also, the Welsh will need updating as well

Reply:
It's a fair point :) thanks @rhianwatkins1 Rhian for flagging the missing the Welsh language change to the cy.yaml, my bad :)

The change to the offline description " Web Chat is available" was approved by the offline team but I'll admit it's a bit ...misleading when I look at it again now.

How does this sound as alternatives?:-
"Sorry, web chat is currently offline, our opening hours are"
or
"Sorry, web chat is currently offline, please see our opening hours listed below"

Reply:
Probably best get Ed's input on correct copy, as it'll need to be translated also

Reply:
ok, I've run through the options with Ed and agreed "Sorry, web chat is currently offline, our opening hours are:" will be acceptable :)
I'll get the content team to provide a validated Welsh translation then update the PR
thanks